docs/admin/dns.md

```diff
@@ -182,6 +182,111 @@ Address 1: 10.0.0.1
 
 If you see that, DNS is working correctly.
 
+### Troubleshooting tips.
+If above `nslookup` command does not work, here are some hints:
+
+#### Check the local DNS configuration first
+Take a look inside the resolv.conf file. (See "Inheriting DNS from the node" and "Known issues" sections for more information)
+```
+cat /etc/resolv.conf
+```
+
+You should see the search path and nameserver are set up like below: (search path may vary for different cloud providers)
```

Comment:
"Verify that the search path and name server are set up like the following (note that seach path may vary for different cloud providers):"